regl-camera.js

```diff
@@ -16,7 +18,13 @@ function createCamera (regl, props_) {
     phi: props.phi || 0,
     distance: Math.log(props.distance || 10.0),
     eye: new Float32Array(3),
-    up: new Float32Array(props.up || [0, 1, 0])
+    up: new Float32Array(props.up || [0, 1, 0]),
+    fovy: props.fovy || Math.PI / 4.0,
+    near: typeof props.near !== 'undefined' ? props.near : 0.01,
+    far: typeof props.far !== 'undefined' ? props.far : 1000.0,
+    flipY: !!props.flipY,
```

Comment:
When I render out in Node.js environment and encode to PNG/JPEG etc. the image is upside down. This is a quick means of flipping the projection vertically.